### PR TITLE
Refactor magic strings to Shim class

### PR DESCRIPTION
**Pull request by bancer**

### CODE REVIEW

File: Config/bootstrap.php [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 // For CakeDC search plugin
 Configure::write('Search', [
-		'Prg' => [
-			'commonProcess' => ['paramType' => 'querystring', 'filterEmpty' => true],
-			'presetForm' => ['paramType' => 'querystring']
-		]
+	'Prg' => [
```
Comment (dereuromark) [inline Config/bootstrap.php]: Indentation looks off.

Comment (bancer) [reply Config/bootstrap.php]: There were 2 tabs here. I removed one.

Comment (dereuromark) [reply Config/bootstrap.php]: Technically, 2 are correct :)
This is on 2nd nesting level.
Build the normalized form and you will see.

**Comment by dereuromark** [comment]
I wish people were so passionate about the 3.x version :P

**Pull request merged**